### PR TITLE
Add bird traits from #52

### LINKED ISSUE #52: Add some new qualities
Add terms 

In 2018, @diatomsRcool started adding these:

2a1f349b-77d6-424b-8fff-a4f37ae54941,head width,PATO:0000921,width,UBERON:0000033,head
b948f71a-e90b-4b9f-b423-f2b1de402742,metafemur length,PATO:0000122,length,HAO:0001140,metafemur
b1990c0f-0437-42e1-98f5-1b47a4387e8a,beak length,PATO:0000122,length,UBERON:0005094,beak
f7e2a5e5-aab3-4cf4-b66e-df6cc883a9f6,cell length,PATO:0000122,length,GO:0005623,cell
4d6b5610-d7c6-4999-bcac-b83bf98d92b4,cell width,PATO:0000921,width,GO:0005623,cell
5c720f5c-61d7-4702-b133-86715dab7aae,fore wing length,PATO:0000122,length,HAO:0000351,fore wing
db2376a8-e854-4da9-81e3-cd9794b889c4,hind wing length,PATO:0000122,length,HAO:0000400,hind wing
f611d111-b792-4128-aa6f-aae2b766ab3d,seed volume,PATO:0000918,volume,PO:0009010,seed
f53b7788-ffa5-4fd1-b97e-e4da787ccf24,testis position,PATO:0000140,position,UBERON:0000473,testis
7f49497f-b27b-47a5-89b2-78aa636fc151,shelled egg length,PATO:0000122,length,UBERON:0007379,shelled egg
4001ad0f-2747-4e06-b95e-f1a22da090b9,shelled egg width,PATO:0000921,width,UBERON:0007379,shelled egg

can we please add them to entity_attribute.tsv (give them proper ids as well)?

Thank you. 

Here is the OLD pr I just closed:
https://github.com/obophenotype/bio-attribute-ontology/pull/24

### PR DESCRIPTION
**Pull request by rays22**
If applied, this commit will fix #52.

### CODE REVIEW

**Comment by matentzn** [comment]
Are there already HAO and PO imports set up on OBA?

It seems to me plant traits may be better served by https://obofoundry.org/ontology/to.html? cc @marieALaporte @cooperl09

**Comment by matentzn** [comment]
Thanks @marieALaporte 
It's probably better if OBA imported TO, or we release some kind of joint product. I perceive OBA to be for metazoans what TO is for plants, but we should work our a better integration in a separate issue. For now, @rays22 can you just remove the TO terms from OBA and make issues n the TO repo to add them?

File: src/patterns/data/default/entity_attribute.tsv [hunk 1]
```diff
@@ -7242,3 +7242,12 @@ OBA:1001089	multicellular organism sex	UBERON:0000468	multicellular organism	PAT
 OBA:1001090	feather color	UBERON:0000022	feather	PATO:0000014	color
 OBA:1001091	egg shell trait	UBERON:0005079	eggshell	PATO:0000001	quality
 OBA:1001092	egg shell color	UBERON:0005079	eggshell	PATO:0000014	color
+OBA:2040143	metafemur length	HAO:0001140	metafemur	PATO:0000122	length
+OBA:2040144	beak length	UBERON:0005094	beak	PATO:0000122	length
+OBA:2040145	cell length	GO:0005623	cell	PATO:0000122	length
+OBA:2040146	cell width	GO:0005623	cell	PATO:0000921	width
+OBA:2040147	fore wing length	HAO:0000351	fore wing	PATO:0000122	length
+OBA:2040148	hind wing length	HAO:0000400	hind wing	PATO:0000122	length
+OBA:2040149	seed volume	PO:0009010	seed	PATO:0000918	volume
```
Comment (matentzn) [inline src/patterns/data/default/entity_attribute.tsv:7251]: remove this. this has to go into PO. Just drop, no further action required.

File: src/patterns/data/default/entity_attribute.tsv [hunk 2]
```diff
@@ -7242,3 +7242,12 @@ OBA:1001089	multicellular organism sex	UBERON:0000468	multicellular organism	PAT
 OBA:1001090	feather color	UBERON:0000022	feather	PATO:0000014	color
 OBA:1001091	egg shell trait	UBERON:0005079	eggshell	PATO:0000001	quality
 OBA:1001092	egg shell color	UBERON:0005079	eggshell	PATO:0000014	color
+OBA:2040143	metafemur length	HAO:0001140	metafemur	PATO:0000122	length
```
Comment (matentzn) [inline src/patterns/data/default/entity_attribute.tsv:7245]: Remove, make uberon issue requesting metafemur, make uberon PR adding metafemur, xrefing to HAO.

Introduce policy: NO species specific anatomy ontology references in OBA. Only UBeron.

File: src/patterns/data/default/entity_attribute.tsv [hunk 3]
```diff
@@ -7242,3 +7242,12 @@ OBA:1001089	multicellular organism sex	UBERON:0000468	multicellular organism	PAT
...
+OBA:2040144	beak length	UBERON:0005094	beak	PATO:0000122	length
+OBA:2040145	cell length	GO:0005623	cell	PATO:0000122	length
+OBA:2040146	cell width	GO:0005623	cell	PATO:0000921	width
+OBA:2040147	fore wing length	HAO:0000351	fore wing	PATO:0000122	length
```
Comment (matentzn) [inline src/patterns/data/default/entity_attribute.tsv:7249]: Add to uberon same as above

File: src/patterns/data/default/entity_attribute.tsv [hunk 4]
```diff
@@ -7242,3 +7242,12 @@ OBA:1001089	multicellular organism sex	UBERON:0000468	multicellular organism	PAT
...
+OBA:2040148	hind wing length	HAO:0000400	hind wing	PATO:0000122	length
```
Comment (matentzn) [inline src/patterns/data/default/entity_attribute.tsv:7250]: Add to uberon same as above

**Review by matentzn — CHANGES_REQUESTED** [review]

File: src/patterns/data/default/entity_attribute.tsv (hunk 1 above)
Comment (rays22) [reply src/patterns/data/default/entity_attribute.tsv:7250]: Done.

File: src/patterns/data/default/entity_attribute.tsv (hunk 4 above)
Comment (rays22) [reply src/patterns/data/default/entity_attribute.tsv]: UBERON has a declared focus on vertebrates. HAO terms do not really fit that scope (Hymenoptera are insects, not vertebrates).
I am removing these Hymenoptera anatomy classes for now:
HAO:0001140	metafemur
HAO:0000351	fore wing
HAO:0000400	hind wing

**Review by matentzn — APPROVED** [review]
Good call @rays22

**Pull request merged**